app/Nova/User.php

```diff
@@ -171,6 +171,13 @@ public function fields(NovaRequest $request): array
                 ->onlyOnDetail()
                 ->hideFromDetail(static fn (NovaRequest $r, AppModelsUser $u): bool => $u->is_service_account),
 
+            Text::make(
+                'Graduation Semester',
+                static fn (AppModelsUser $u): string => $u->getHumanReadableSemesterAttribute()
+            )
```

Comment:
Because this is now a computed attribute on the model, it does not need to be a computed field on the Nova side.

```suggestion
            Text::make('Graduation Semester', 'human_readable_graduation_semester')
```